### DISCUSSION TITLE
Some notifications missing 'dismiss' button

### DISCUSSION THREAD

**Discussion by cacarr-pdxweb** [linuxdeepin/developer-center#3055] [Features Request & Ideas | 特性请求 & 头脑风暴]
Notifications from Chrome sometimes do not have a 'dismiss' button. You are forced to click through the notification to dismiss.

**Comment by justforlxz** [comment]
Confirmed with dde's product manager that there is no dedicated close button.

I assigned the issue to the community product manager and waited for him to negotiate with the product manager of dde. Thank you!
